atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1008,7 +1008,7 @@ private static LockAndTimestampServices createRawLeaderServices(
                 .sslConfiguration(leaderConfig.sslConfiguration())
                 .build();
         ServiceCreator creator = ServiceCreator.noPayloadLimiter(
-                metricsManager, () -> serverListConfig, userAgent, () -> RemotingClientConfig.DEFAULT);
+                metricsManager, () -> serverListConfig, userAgent, () -> RemotingClientConfig.ALWAYS_USE_LEGACY);
```

Comment:
Don't we want conjure here?

Reply:
Not quite. This would be used by the embedded leader mode, which is still used in certain installations of internal auth product. I'm not as confident we have good test coverage for the way CJR works in the context of such situations - we'll need to do this soon, but didn't see this as in scope for this specific PR.